src/odemis/gui/util/img.py

```diff
@@ -860,24 +860,28 @@ def calculate_ticks(value_range, client_size, orientation, tick_spacing):
     vtp_ratio = abs(pixel_space / value_space)  # px/unit
     epsilon = (1 / vtp_ratio) / 10  # "tiny" value: a 10th of a pixel
 
-    num_ticks = pixel_space // tick_spacing
+    # Find the ticks to show so that it looks good for the user. It must have a
+    # constant spacing between the ticks, and this spacing should be a "easy
+    # number": 1, 2, 5, 7.5, or any of these values multiplied by a power of 10.
+    # For example, 150 is not good, but 200 is fine.
+
+    # minimum number of ticks that we want
+    num_ticks = max(2, pixel_space // tick_spacing + 1)
```

Comment:
Is there a testcase where you test for 'uneasy' values? in case the coding changes in the future and someone messes up this might be handy.

Reply:
The test case "test_simple" would trigger the infinite loop bug. What do you mean by "uneasy" values? Maybe the name of the test case is not so well chosen, but it contains quite a few different inputs, which could trigger different issues if not handled properly.